plugins/inputs/filecount/filecount.go

```diff
@@ -14,6 +14,11 @@ import (
 
 const sampleConfig = `
   ## Directory to gather stats about.
+  ## This accept standard unit glob matching rules, but with the addition of
+  ## ** as a "super asterisk". ie:
+  ##   /var/log/**    -> recursively find all directories in /var/log
+  ##   /var/log/*/*   -> find all directories with a parent dir in /var/log
+  ##   /var/log       -> only count the /var/log dir
```

Comment:
In this context, "only count the /var/log dir" is a bit misleading. By default, using `/var/log` counts all files in `/var/log` and all of its subdirectories.

In my opinion we should distinguish counting files in a directory from producing separate stats (fields) for a directory.